src/Mod/Start/StartPage/StartPage.py

```diff
@@ -789,9 +790,13 @@ def exportTestFile():
 
     "Allow to check if everything is Ok"
 
-    f = open(os.path.expanduser("~") + os.sep + "freecad-startpage.html", "w")
-    f.write(handle())
-    f.close()
+    with codecs.open(
+        os.path.expanduser("~") + os.sep + "freecad-startpage.html",
+        encoding="utf-8",
```

Comment:
Will this always be UTF-8?

Reply:
It's considered a best practice in Python to always explicitly open files (for reading _and_ writing) with the encoding set. If that file might *not* be UTF-8, the fix is to ensure that the corresponding `open(..., "w")` statement has the encoding parameter set as well.